events/belgium/folkbalbende.yaml

```diff
@@ -13773,3 +13773,18 @@ events:
     social: false
     price: €20-€55
     organisation: Frisse Folk Vzw/asbl
+
+  - name: Danslab Folk
+    details: "Tijdens de folkdanslessen voor beginners leer je de basis en variaties van de typische folkdansen, zoals bijvoorbeeld wals, scottish, bourrée en tovercirkel.\r\nBij de gevorderden gaan we dieper in op deze en andere dansen (branle d'Ossau, avant-deux, fandango, ...)\r\n\r\nAls leerling-danser heb je 2.5u les per week.\r\nIn het uurtje folkdanslab zetten we wat we geleerd hebben in de les om in praktijk, en dansen we leuke groepsdansen."
+    links:
+      - "https://www.schoten.be/academie/dans/danslab-folk"
+    start: "2023-09-05T18:00:00+02:00"
+    end: "2024-06-25T22:00:00+02:00"
```

Comment:
This start and end time doesn't look correct. Please add individual events, not a single entry for a whole series.